scanner/updater/export.go

```diff
@@ -120,32 +120,48 @@ func Export(ctx context.Context, outputDir string) error {
 		},
 	}
 
-	dataStore, err := jsonblob.New()
-	mgr, err := updates.NewManager(ctx, dataStore, updates.NewLocalLockSource(), httpClient,
+	zstdWriter, err := zstd.NewWriter(outputFile)
+	if err != nil {
+		return err
+	}
+	defer zstdWriter.Close()
+
+	updaterStore, err := jsonblob.New()
```

Comment:
@RTann  I am using two `updaterStore, err := jsonblob.New()` and  `configStore, err := jsonblob.New() `because one dataStore (jsob blob) will consume two much memory and sometimes kills the workflow based on my testing experiences. And I am using two update managers accordingly.